src/main/java/ui/windows/StatsDelegator.java

```diff
@@ -1,14 +1,19 @@
 package ui.windows;
 
+import database.CSVDatabase;
+import usecases.PlayerInformation;
+
 import javax.swing.*;
 import java.awt.event.ActionListener;
 import java.awt.event.ActionEvent;
+import java.io.IOException;
 
 /**
  * Delegates incoming user input from a StatsDisplay.
  */
 public class StatsDelegator implements ActionListener {
     private StatsController controller;
+    private CSVDatabase database = new CSVDatabase();
 
```

Comment:
CSV database should not be instantiated here, instead, it should be instantiated in Main as a DataAccess object, and passed it into the StatsDelegator controller (dependency injection).